apps/switchboard/src/server.ts

```diff
@@ -133,44 +135,47 @@ async function initServer(serverPort: number, options: StartServerOptions) {
       ? ".ph/read-storage"
       : dbPath;
 
-  const driveServer = new ReactorBuilder([
-    documentModelDocumentModelModule,
-    driveDocumentModelModule,
-  ] as unknown as DocumentModelModule[])
-    .withStorage(storage)
-    .withCache(cache)
-    .withOptions({
-      featureFlags: {
-        enableDualActionCreate:
-          options.reactorOptions?.enableDualActionCreate ?? false,
-      },
-    })
-    .build();
```

Comment:
@acaldas this is the main problem, we need to build the driveServer before passing it to startAPI, but at this point we are not loading local packages, so if the storage contains a document that is not a documentModel or a drive, it will not be able to handle it